src/App.css

```diff
@@ -11,67 +24,60 @@ li {
 }
 
 .nav-wrap p {
-  color: white;
-  font-size: 30px;
-  font-weight: 600;
   margin-left: 15px;
+  font-size: 1.875rem;
+  font-weight: 800;
+  color: rgb(2, 144, 255);
+}
+
+.book-list {
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.book-list li {
+  width: 100%;
+  padding: 10px;
 }
 
 .nav-wrap ul {
   display: flex;
   justify-content: flex-end;
-  margin-right: 40px;
+}
+
+.Book-container li {
+  display: flex;
+  align-items: center;
 }
 
 .nav-wrap ul li {
   list-style: none;
   margin: 10px 30px;
-  color: white;
+  color: black;
 }
 
 .book-page {
   width: 100%;
-  background-color: yellowgreen;
-  height: 100vh;
+  height: 100%;
   display: flex;
   justify-content: center;
-}
-
-.book-form {
-  display: flex;
-  flex-direction: column;
+  flex-direction: column-reverse;
   align-items: center;
-  justify-content: center;
-}
-
-.entry {
-  height: 34px;
-  border-radius: 10px;
-  border: 3px solid yellow;
-  margin: 10px;
-  width: 500px;
-  color: darkgray;
-  padding: 5px;
-  font-size: 20px;
-}
-
-.submit-btn {
-  background-color: gray;
-  color: yellow;
-  font-size: 20px;
-  cursor: pointer;
-  width: 520px;
-  height: 40px;
-  border-radius: 10px;
-  border: 3px solid yellow;
 }
 
 .Book {
+  width: 90%;
+  box-shadow: rgba(0, 0, 0, 0.15) 0 2px 8px;
   background-color: white;
-  padding: 3px 15px;
+  margin-left: auto;
+  margin-right: auto;
   display: flex;
-  align-items: center;
   justify-content: space-between;
+  align-items: center;
+  padding: 10px;
```

Comment:
- [ ] According to the project requirement, Styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). So I would suggest you please use the same color, font and to match the original design. See the design below for reference
- [ ] I have to underline below so please use the same color and font and also fix the spacing provided on the design. 
### Your design
![his](https://user-images.githubusercontent.com/81727988/222903658-393fc09b-bd86-495b-8147-ab00a5821f63.png)


### Figma design
![original](https://user-images.githubusercontent.com/81727988/222903663-c9aac9e0-54fc-4de4-9d7f-5796f97c86b6.png)